# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/libmspub/hotfix-gcc15.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- libmspub-0.1.4/src/lib/Coordinate.h.vanilla	2025-04-25 23:41:02.462071176 +0200
+++ libmspub-0.1.4/src/lib/Coordinate.h	2025-04-25 23:41:13.289866217 +0200
@@ -11,6 +11,7 @@
 #define INCLUDED_COORDINATE_H
 
 #include "MSPUBConstants.h"
+#include <cstdint>
 
 namespace libmspub
 {
